testing-app/src/pages/FAQPage.tsx

```
@@ -0,0 +1,68 @@
+            <h1 className="faq-page-title">Preguntas Frecuentes</h1>
+            <p className="faq-page-subtitle">
+              Aquí encontrarás respuestas a las preguntas más comunes sobre la pagina de Ithaka.
+            </p>
+            <div className="faq-list">
+              {faqData.map((item, index) => (
+                <div key={index} className="faq-item">
+                  <h3
+                    className="faq-question"
+                    onClick={() => toggleFAQ(index)}
+                  >
+                    {item.question}
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)